tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -573,8 +580,17 @@ private static void SetupGithubActionsEnvironment()
             PipelineId = EnvironmentHelpers.GetEnvironmentVariable("GITHUB_RUN_ID");
             PipelineNumber = EnvironmentHelpers.GetEnvironmentVariable("GITHUB_RUN_NUMBER");
             PipelineName = EnvironmentHelpers.GetEnvironmentVariable("GITHUB_WORKFLOW");
-            PipelineUrl = $"https://github.com/{EnvironmentHelpers.GetEnvironmentVariable("GITHUB_REPOSITORY")}/commit/{Commit}/checks";
-            JobUrl = PipelineUrl;
+            var attempts = EnvironmentHelpers.GetEnvironmentVariable("GITHUB_RUN_ATTEMPT");
+            if (string.IsNullOrWhiteSpace(attempts))
+            {
+                PipelineUrl = $"{serverUrl}/{EnvironmentHelpers.GetEnvironmentVariable("GITHUB_REPOSITORY")}/actions/runs/{PipelineId}";
```

Comment:
Looks like you can use `Repository` instead of calling `GetEnvironmentVariable()` again.

```suggestion
                PipelineUrl = $"{serverUrl}/{Repository}/actions/runs/{PipelineId}";
```

Reply:
done in dfa058e, I had to modify the suggestion a little bit.